### ISSUE TITLE
Hide post on my dash

### ISSUE THREAD

**Issue by xThreeh** [new-xkit/XKit#1143]
Hide post that my followers rebloged of my blog

spanish:
Quiero ocultar los post que me rebloguean , existe esta opción actualmente? 
ejemplo:
![screenshot_1](https://cloud.githubusercontent.com/assets/16139465/18059794/11b919d4-6df1-11e6-924f-d577c4420be1.png)
hide this post :arrow_up:

**Comment by ThePsionic** [comment]
Not possible at the moment, but sounds like something that we could integrate in Servant.

**Comment by nightpool** [comment]
@ThePsionic @Franco Not sure I understand the request—you want to hide
posts that people you follow reblog from you? What if they add a comment?

**Comment by ThePsionic** [comment]
Correct. If they add a comment it should still be visible on the blog page and activity page as normal, except hidden on the dashboard, I imagine. Also not sure why you tagged Franco instead of @xThreeh - maybe because that's his real name :stuck_out_tongue:

**Comment by nightpool** [comment]
Yeah, sorry, when replying from email I can't see the usernames, just the
realnames

**Comment by xThreeh** [comment]
@ThePsionic @nightpool Yes, i want to hide
posts that people me follow reblog from me.
